src/test-utils/factories/transactionOutcome.factory.ts

```diff
@@ -0,0 +1,18 @@
+import { TransactionOutcome } from "../../types";
+
+const DEFAULT_TRANSACTION_OUTCOME: TransactionOutcome = {
```

Comment:
What is this transaction outcome for? Zap? Deposit?

Reply:
`TransactionOutcome` is used for a lot of things in the `SimulationInterface` as we don't have more specific types for each transaction. I've just created this factory so that the tests are more realistic. Is there any action here Fibo?